jetcd-examples/jetcd-simple-ctl/README.MD

```diff
@@ -4,46 +4,50 @@ Download and start an etcd node from [instruction](https://github.com/coreos/etc
 
 > Note: For windows machine, run `.\mvnw.cmd` instead of `./mvnw`
 
-Build the example jar.
+Build the example jar with provided [maven wrapper][mavenwrapper].
```

Comment:
Build the example jar `target/jetcdctl.jar` with the provided [maven wrapper][mavenwrapper]: